src/components/App.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  CssBaseline,
+  createStyles
```

Comment:
sort within imported components (lines 3 to 7)

Reply:
Sounds right

Reply:
Resolved itself when I changed rules for max line length and import order